lib/workers/repository/onboarding/branch/index.ts

```diff
@@ -21,7 +29,27 @@ export async function checkOnboardingBranch(
   logger.debug('checkOnboarding()');
   logger.trace({ config });
   let onboardingBranch = config.onboardingBranch;
-  const repoIsOnboarded = await isOnboarded(config);
+  let repoIsOnboarded: boolean;
+  const onboardingBranchSha = getBranchCommit(onboardingBranch!);
+  const baseBranchSha = getBranchCommit(config.baseBranch!);
```

Comment:
i think this can only be default branch 
```suggestion
  const baseBranchSha = getBranchCommit(config.defaultBranch!);
```